webgrid/renderers.py

```diff
@@ -188,6 +192,62 @@ def render_attr(key, value):
     return Markup(' ' + ' '.join(rendered_attrs))
 
 
+class JSON(Renderer):
+    """Renderer for JSON output"""
+
+    @property
+    def name(self):
+        return 'json'
+
+    def serialized_columns(self):
+        return {col.key: str(col.label) for col in self.columns}
+
+    def serialize_filter(self, filter):
+        return types.Filter(
+            op=filter.op,
+            value1=filter.value1,
+            value2=filter.value2,
```

Comment:
Should these use `value1_set_with` and `value2_set_with` so that the filter's displayed value does not change based on processing? That appears to be how we set the input values for the HTML renderer